.github/workflows/auto-comment-pr.yml

```
@@ -19,7 +19,11 @@ jobs:
           github-token: ${{ secrets.BOT_TOKEN  }}
           script: |
             const { owner, repo, number: issue_number } = context.issue;
-            const commentBody = "🚀Thanks for your contribution🎉. CodeRabbit(AI) will review your code first🔥";
+            
+            // Get PR creator's username
+            const prCreator = context.payload.pull_request.user.login;
+            
```

yamllint

[error] trailing spaces

(trailing-spaces)